nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/main/java/org/apache/nifi/controller/repository/StandardProcessSession.java

```diff
@@ -2527,6 +2554,43 @@ public String toString() {
         return "StandardProcessSession[id=" + sessionId + "]";
     }
 
+    @Override
+    public String getUnacknowledgedFlowfileInfo() {
```

Comment:
The rollback that logs this occurs in AbstractProcessor, delegating to the session to provide the logging information.  If I understand what you're saying, the logging should be moved directly into the Session rollback methods, is that correct?